less/select.less

```diff
@@ -22,8 +22,8 @@
 @select-input-border-width:         1px;
 @select-input-height:               36px;
 @select-input-internal-height:      (@select-input-height - (@select-input-border-width * 2));
-@select-input-placeholder:          #aaa;
-@select-text-color:                 #333;
```

Comment:
Please make it only for the unique input, as this affects to all inputs
After this the text looks more darker

| Before | After |
| --- | --- |
| ![screen shot 2018-11-26 at 10 41 22 am](https://user-images.githubusercontent.com/1847316/49028437-07874b00-f168-11e8-97df-d599bf6ff00e.png) |  ![screen shot 2018-11-26 at 10 41 33 am](https://user-images.githubusercontent.com/1847316/49028435-07874b00-f168-11e8-8865-0763edb5d121.png) |

Reply:
Yeah, I talked to UX team about that and they are expecting to do that change on all of the selects